next-app/src/routes/password/change/+page.svelte

```diff
@@ -1,39 +1,36 @@
-<script>
-	import { goto } from '$app/navigation'
-	import { UPDATE } from '$lib/fetch'
+<script lang=ts>
 	import {
 		Button,
 		Form,
 		Input,
 	} from '$lib/forms'
 	import PageHeader from '$lib/PageHeader.svelte'
+	import { error } from '$lib/error'
+    import type { ActionData } from './$types'
```

Comment:
The indentation here is spaces. Everything else is tabs 🤔

Reply:
probably an auto-import, I'll have to check and see if that's configurable.  At any rate, thank you, corrected in [f0b8a45](https://github.com/sillsdev/web-languageforge/pull/1716/commits/f0b8a45bacb4ffa1296ec880c7d3473becbdf47f)

Reply:
we should revisit Prettier or another code formatter for Svelte so that we are not dealing with these types of changes, or have inconsistencies like tabs and spaces.